values.yaml

```diff
@@ -474,3 +474,12 @@ redis:
     password: ""
   ## Additional deployment annotations
   podAnnotations: {}
+
+jobConfig:
+  # if enabled setup proper Harbor settings
+  # https://github.com/goharbor/harbor/blob/master/docs/configure_user_settings.md#harbor-user-settings
+  enabled: false
+  settings: {}
+  image:
+    repository: byrnedo/alpine-curl
```

Comment:
Here unfortunately I wasn't able to find an "official" curl docker image.

Tell us how to move forward with an official image hosted by goharbor per example.